arcgis-ios-sdk-samples/Route & Directions/Navigate route/NavigateRouteViewController.swift

```diff
@@ -13,10 +13,14 @@
 // limitations under the License.
 
 import UIKit
-import ArcGIS
 import AVFoundation
```

Comment:
FYI: It looks like this import isn't needed.

```suggestion
```

Reply:
Interesting. In the sample I used `AVSpeechSynthesizer` from [`AVFoundation`](https://developer.apple.com/documentation/avfoundation/avspeechsynthesizer), but after removing the import it still compiles with no error. 

When should I explicitly import a framework?

Reply:
Never mind. Even though it compiles without it (I'm not sure why), the file does rely on it, so leave it.